run_historical_conf.py

```diff
@@ -26,31 +27,26 @@ def new_camera(camera, camera_index, config_file):
         new_config.write(add_configfile)
     add_configfile.close()  
```

Comment:
The cose sentence is not required because you open the file in a `with` block.

Reply:
Done